cognite/neat/rules/models/_rules/dms_schema.py

```diff
@@ -35,6 +45,14 @@ class DMSSchema:
     containers: dm.ContainerApplyList = field(default_factory=lambda: dm.ContainerApplyList([]))
     node_types: dm.NodeApplyList = field(default_factory=lambda: dm.NodeApplyList([]))
 
+    _FIELD_NAME_BY_RESOURCE_TYPE: ClassVar[dict[str, str]] = {
```

Comment:
why not enum class? we are already using it in many locs across the code base?

Reply:
It is an implementation detail for this class, only pulled out for reuse in from_zip and from_directory.